cardano-node/src/Cardano/Node/Configuration/Logging.hs

```diff
@@ -211,3 +232,56 @@ createLoggingLayer ver nodeConfig' = do
 
 shutdownLoggingLayer :: LoggingLayer -> IO ()
 shutdownLoggingLayer = shutdown . llSwitchboard
+
+-- The node provides the basic node's information for TraceForwarderBK.
+-- It will be sent once TraceForwarderBK is connected to an external process
+-- (for example, RTView).
+nodeBasicInfo :: NodeConfiguration
+              -> Consensus.TopLevelConfig (CardanoBlock c)
+              -> UTCTime
+              -> IO [LogObject Text]
+nodeBasicInfo nc cfg nodeStartTime' = do
+  let CardanoLedgerConfig cfgByron cfgShelley cfgAllegra cfgMary = Consensus.configLedger cfg
+      protocolDependentItems =
+        case ncProtocol nc of
+          ByronProtocol ->
+             let genesis = byronLedgerConfig cfgByron
+             in [ ("systemStartTime",  show (getSystemStart $ Consensus.configBlock cfg))
```

Comment:
It doesn't compile. @mrBliss could you help me please?

Reply:
> It doesn't compile.
What's the error?

Reply:
@mrBliss Now I see another error:

```
      Expected type: Consensus.TopLevelConfig (Consensus.CardanoBlock c)
        Actual type: Consensus.TopLevelConfig blk
```